docs/aerodromes/classd/Bankstown.md

```diff
@@ -139,4 +161,14 @@ VFR aircraft require a level readback.
     <span class="hotline">**BK ADC** -> **SY TCU**</span>: "UJN, A010"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
```

Comment:
This is how we have it to be standardised with all other Class D towers. Essentially so the tower knows that all coordinated IFR arrivals have been cleared for the approach. Would this cause any issues at YSBK?

Reply:
99% of IFR arrivals are from Class G at BK rather than CTA. So, control instruction to an Acft OCTA, probably doesn't work. 

However, an Acft does require clearance for an INST APCH prior to handoff.

This differs from somewhere like PF, where 99% of IFR arrivals are via PAL, and call PF TWR assigned A015V day / A018 night (or cleared INST APCH if applicable.

APP/ENR issuing VSA's removes TWR's ability to use levels as a form of Separation/Segregation.

Not 100% sure we can actually standardize this between units, I'm sure other towers will also differ.